onnxruntime/contrib_ops/cpu/quantization/matmul_nbits.cc

```diff
@@ -20,30 +20,158 @@ class MatMulNBits final : public OpKernel {
         K_{narrow<size_t>(info.GetAttr<int64_t>("K"))},
         N_{narrow<size_t>(info.GetAttr<int64_t>("N"))},
         block_size_{narrow<size_t>(info.GetAttr<int64_t>("block_size"))},
-        nbits_{narrow<size_t>(info.GetAttr<int64_t>("bits"))} {
+        nbits_{narrow<size_t>(info.GetAttr<int64_t>("bits"))},
+        accuracy_level_{info.GetAttr<int64_t>("accuracy_level")} {
```

Comment:
At present, the `accuracy_level` only supports three values: 0 (unset), 1 (fp32), and 4 (int8). To avoid executing an invalid `MatMulNBits` node, we should add a `ORT_ENFORCE` for verification, similar to the existing `nbits_` check.

Reply:
@yihonglyu An Unsupported accuracy level should fall through to a supported higher accuracy level, e.g. fp16 falls through to fp32.